### DISCUSSION TITLE
[VIPS.warning] IDAT: ADLER32 checksum mismatch

### DISCUSSION THREAD

**Discussion by zhifengzhuang** [libvips/libvips#2930] [Ideas]
A lot of this error messages are printed. The number of message is many times the number of business requests.I notice that there is a fix in Version 1.6.27rc01.
<img width="789" alt="image" src="https://user-images.githubusercontent.com/13492988/179349161-3bea4d6f-a590-4137-b11e-c9f7f308fc7f.png">
I don't know if PNG_IGNORE_ADLER32 can solve this problem.Is necessary to support this png option in libvips?

**Comment by jcupitt** [comment]
Hi @zhifengzhuang,

Good idea, let's turn this option off if we can. See https://github.com/libvips/libvips/commit/57fff0fe2b7335eb116cb6f83b92dfa02f883c58

**Reply by zhifengzhuang** [reply]
👍
